mxcube3/ho_mediators/beamline_setup.py

```diff
@@ -268,7 +268,7 @@ def set(self, value):
         :rtype: float
         """
         try:
-            self._ho.startMoveEnergy(float(value))
+            self._ho.start_move_energy(float(value))
```

Comment:
I've updated this on my side to, alltough I have not made the PR yet.  In any ways not very important 
You should perhaps also pass wait=false to not block the UI